language/move-lang/stdlib/modules/libra_account.move

```diff
@@ -102,113 +150,203 @@ module LibraAccount {
     // However those account can also mint to themselves so that is a decent workaround
     public fun mint_to_address(payee: address, amount: u64) acquires T {
         // Create an account if it does not exist
-        if (!exists(payee)) create_new_account(payee, 0);
+        if (!exists(payee)) {
+            create_account(payee);
+        };
 
         // Mint and deposit the coin
         deposit(payee, LibraCoin::mint_with_default_capability(amount));
     }
 
+    // Helper to withdraw `amount` from the given `account` and return the resulting LibraCoin::T
+    fun withdraw_from_account(account: &mut T, amount: u64): LibraCoin::T {
+        LibraCoin::withdraw(&mut account.balance, amount)
+    }
+
     // Withdraw `amount` LibraCoin::T from the transaction sender's account
     public fun withdraw_from_sender(amount: u64): LibraCoin::T acquires T {
         let sender_account = borrow_global_mut<T>(Transaction::sender());
-        // Abort if sender has delegated the privilege to withdraw from her account elsewhere.
-        if (sender_account.delegated_withdrawal_capability) abort 11;
-
-        // The sender has retained her withdrawal privileges
-        LibraCoin::withdraw(&mut sender_account.balance, amount)
+        // The sender has delegated the privilege to withdraw from her account elsewhere--abort.
+        Transaction::assert(!sender_account.delegated_withdrawal_capability, 11);
+        // The sender has retained her withdrawal privileges--proceed.
+        withdraw_from_account(sender_account, amount)
     }
 
     // Withdraw `amount` LibraCoin::T from account under cap.account_address
     public fun withdraw_with_capability(
-        cap: &WithdrawalCapability,
-        amount: u64,
-    ): LibraCoin::T
-        acquires T
-    {
-        LibraCoin::withdraw(&mut borrow_global_mut<T>(cap.account_address).balance, amount)
+        cap: &WithdrawalCapability, amount: u64
+    ): LibraCoin::T acquires T  {
+        let account = borrow_global_mut<T>(cap.account_address);
+        withdraw_from_account(account, amount)
     }
 
     // Return a unique capability granting permission to withdraw from the sender's account balance.
     public fun extract_sender_withdrawal_capability(): WithdrawalCapability acquires T {
-        let account_address = Transaction::sender();
-        let sender_account = borrow_global_mut<T>(account_address);
-        // Abort if wWe already extracted the unique withdrawal capability for this account.
-        if (sender_account.delegated_withdrawal_capability) abort 11;
+        let sender = Transaction::sender();
+        let sender_account = borrow_global_mut<T>(sender);
+
+        // Abort if we already extracted the unique withdrawal capability for this account.
+        Transaction::assert(!sender_account.delegated_withdrawal_capability, 11);
 
-        // ensure uniqueness of the capability
+        // ensure the uniqueness of the capability
```

Comment:
Minor nit: This comment should be capitalized to match the style elsewhere in this file